frontend/views/containers/modals/DesignSystem.vue

```diff
@@ -1,5 +1,12 @@
 <template lang="pug">
+//- NOTE: This file is here to serve as a test case / example
+//-       In my (greg's) opinion, it should be deleted soon.
```

Comment:
Really? I think absolutely the opposite. I see this file as an entry point for any frontend developer to this project to get quickly familiar with our Design System and CSS framework.

Reply:
OK, I just have a feeling it's going to diverge from actual usage and that might be confusing. I was thinking that maybe the actual usage (example modals throughout the project) might serve as better examples...

Reply:
Personally having "demos" is always a good entry point to any feature. That's the purpose of DS page, to be a "demo" of every component we have (not only modals, but also forms, messages, etc...)